sdk/pbs/controller/api/resources/groups/group.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
+// 	protoc-gen-go v1.28.1
```

Comment:
Can you update the tools file if we're going to standardize on this for the next release?

Reply:
The tools file doesn't contain versions, only modules, right? The version is already controlled by `go.mod`, which is updated by this PR.